hs/SvgParsing/Generator.hs

```diff
@@ -39,12 +39,12 @@ makeSVGDoc :: M.Map String String
            -> Bool    -- ^ Whether to include inline styles in the graph.
            -> S.Svg   -- ^ The completed SVG document.
 makeSVGDoc courseMap rects ellipses edges regions regionTexts styled =
-    S.docTypeSvg ! A.width "1052.3622"
-                 ! A.height "744.09448"
+    S.docTypeSvg ! A.width "1195"
```

Comment:
Mmm. In the future, this should be changed to be based off of the width/height of input graph.